packages/fiori/src/themes/UploadCollection.css

```diff
@@ -0,0 +1,123 @@
+:host {
```

Comment:
consider the "hidden" attribute, use some of the existing components for reference, but in general if the someone sets "hidden" to the component (to the host) the component should hide

Reply:
Done